spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -665,7 +801,7 @@ not be written out. More details about the `ItemProcessor` interface can be
 found in <<readersAndWriters.adoc#readersAndWriters,Readers And Writers>>.
 
 
-=== Batch Namespace
+=== Batch Namespace When Using XML Configuration
```

Comment:
I like the idea.  But the header is not rendered properly when placed in the ifdef block.  It loses its formatting :-(